settings/src/main/scala/org/typelevel/sbt/TypelevelSettingsPlugin.scala

```diff
@@ -29,13 +29,17 @@ object TypelevelSettingsPlugin extends AutoPlugin {
   object autoImport {
     lazy val tlFatalWarnings =
       settingKey[Boolean]("Convert compiler warnings into errors (default: false)")
+    lazy val tlJvmRelease =
+      settingKey[Int](
```

Comment:
There's no universe where this somehow can no longer be represented as an `Int` right? 😅